test/fixtures/linter/autofix/ansi/007_with_clause/after.sql

```diff
@@ -1,5 +1,5 @@
 -- Dealing with complicated indents before with clauses.
     	WITH cte as (
             select a from tbla
-        )
+)
         select a from cte
```

Comment:
```suggestion
        select a from cte

```

Reply:
Added the `newline` in both `before.yml` and `after.yml`